test/arguments.js

```diff
@@ -0,0 +1,41 @@
+import test from 'ava';
+import shell from 'shelljs';
+import tempfile from 'tempfile';
+import m from '../';
+
+shell.config.silent = true;
+
+// Test in a temp directory to avoid commits affecting current project in case of bugs
+let oldCwd;
+let tempdir;
+
+test.before(() => {
+	tempdir = tempfile();
+	shell.mkdir(tempdir);
+	shell.cd(tempdir);
+});
+
+test.serial('passing arguments while mocking only commit', async t => {
```

Comment:
Because of https://github.com/stevemao/mock-bin/issues/5. I now realise that this makes absolutely no sense, because the other tests are also affected.